reana_workflow_engine_serial/tasks.py

```diff
@@ -53,7 +53,6 @@ def initialize(workflow_workspace, operational_options):
         cache_enabled = False
 
     # build workspace path
```

Comment:
I think we can delete the comment and a blank line here as well. Also, could you please squash the commits to a single one?